Documentation/ceph-csi-drivers.md

```diff
@@ -129,9 +129,6 @@ kubectl create -f cluster/examples/kubernetes/ceph/csi/example/rbd/storegeclass.
 
 ## Create RBD Secret
 
-Create a Secret that matches `adminid` or `userid` specified in the
-[storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml).
-
 Find a Ceph operator pod (in the following example, the pod is
```

Comment:
(unable to provide a comment above) Should we correct the documentation in this PR to reflect that we need a cluster created before the CSI pods are started? Changes in Rook now make it so that, if only the operator is created then CSI (and other) pods are not started.

Reply:
cluster is pre-req for ceph-csi https://github.com/rook/rook/blob/master/Documentation/ceph-csi-drivers.md#prerequisites